examples/bond/scripts/run/common/common.js

```diff
@@ -68,6 +68,209 @@ async function optInTx (deployer, managerAcc, lsig, assetIndex) {
   await executeTransaction(deployer, optInTx);
 }
 
+/**
+ * Returns group transaction for buy
+ * @param buyer buyer account
+ * @param issuerLsig issuer logic signature
+ * @param amount amount of bonds
+ * @param algoAmount amount of algo
+ * @param appID application index
+ * @param bondID bond index
+ */
+function buyTx (buyer, issuerLsig, amount, algoAmount, appID, bondID) {
```

Comment:
I think we should be able to compute Bond price is fixed. I think we don't need to provide `amount` - we are able to compute it based on `algoAmount`.  If so - let's update the documentation.
If you want we can do it in a separate task -- feel free to create it so we can close this PR earlier and do the new task right after merging this PR.

Reply:
Other idea is to create a second helper function which will compute the amount and then run this function.

Reply:
ok, added a task for it.